Lesson31/Firat/my-app/src/components/NewItem/index.jsx

```diff
@@ -0,0 +1,27 @@
+import "./styles.css";
+import React, { useState } from "react";
```

Comment:
IN the current version of React you don't need to write `import React from "react";`, it's handled by default :)